api/v1/mongodbcommunity_types.go

```diff
@@ -128,6 +132,34 @@ type CustomRole struct {
 	AuthenticationRestrictions []AuthenticationRestriction `json:"authenticationRestrictions,omitempty"`
 }
 
+type Metrics struct {
```

Comment:
Q: Any reason to wrap this in Metrics struct instead of using Prometheus struct directly?

Reply:
No reason in particular, maybe preparing for other options in the future, to go besides Prometheus; but I think that posibility is unlikely.

I don't have a strong opinion. Maybe it is unecessary at this point. I will remove